src/backend/db/migrations/1_initial-migrations.sql

```diff
@@ -4,7 +4,32 @@ CREATE TABLE "user" (
   user_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
   first_name TEXT NOT NULL,
   last_name TEXT NOT NULL,
-  role TEXT NOT NULL CHECK (role IN ('para', 'case_manager', 'admin'))
+  role TEXT NOT NULL CHECK (role IN ('para', 'case_manager', 'admin')),
+  email TEXT UNIQUE NOT NULL,
+  email_verified_at TIMESTAMPTZ,
+  image_url TEXT
+);
+
+CREATE TABLE "account" (
```

Comment:
Please add a comment mentioning the table's use, its relation to the user table, and maybe also indicate this is managed by the NextAuth adapter